src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/Directory.Build.props

```diff
@@ -1,11 +1,13 @@
 <Project>
-<PropertyGroup>
-  <!--
-    The Oniguruma package for regex testing has its own Arcade & SDK independent build mechanisms that automatically look for hierarchical Directory.Build.props files.
-    Therefore, we have this file here in order to suppress the usage of the global SDK and arcade build system to enable yarn install to pass.
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
 
-    For context the Oniguruma contains C++ code that's dynamically compiled on install.
-  -->
-  <_SuppressSdkImports>true</_SuppressSdkImports>
-</PropertyGroup>
+  <PropertyGroup>
+    <!--
+      The Oniguruma package for regex testing has its own Arcade & SDK independent build mechanisms that automatically look for hierarchical Directory.Build.props files.
+      Therefore, we have this file here in order to suppress the usage of the global SDK and arcade build system to enable yarn install to pass.
+
+      For context the Oniguruma contains C++ code that's dynamically compiled on install.
+    -->
+    <_SuppressSdkImports>true</_SuppressSdkImports>
```

Comment:
I _am_ slightly curious if this is still needed, I commented it out, cleared out my node_modules, and `build` was still able to install it fine.
But I really haven't the slightest clue about how all this MSBuild magic works, so I kept it to be safe :).